src/aero_rep_solver.c

```diff
@@ -229,6 +229,51 @@ void aero_rep_get_effective_radius__m(ModelData *model_data, int aero_rep_idx,
   return;
 }
 
+/** \brief Get the surface area of interfacial layer between two phases (m^2)
+ *
+ * Calculates surface area of a interfacial layer (m^2), as well as the set of
+ * \f$\frac{\partial A}{\partial y}\f$ where \f$y\f$ are variables on the
+ * solver state array.
+ *
+ * \param model_data Pointer to the model data
+ * \param aero_rep_idx Index of aerosol representation to use for calculation
+ * \param aero_phase_idx_first Index of the first aerosol phase within the
+ *                       aerosol representation
+ * \param aero_phase_idx_second Index of the second aerosol phase within the
+ *                       aerosol representation
+ * \param surface_area_layer Pointer to surface area of interfacial layer (m^2)
+ * \param partial_deriv Pointer to the set of partial derivatives to be
+ *                      calculated \f$\frac{\partial A}{\partial y}\f$,
+ *                      or a NULL pointer if no partial derivatives are needed
+ */
+void aero_rep_get_interface_layer_surface_area__m2(ModelData *model_data, int aero_rep_idx,
```

Comment:
```suggestion
void aero_rep_get_interface_surface_area__m2(ModelData *model_data, int aero_rep_idx,
```
I would remove layer from this function name. Other aerosol representations may have condensed phases that share a common interface, but are not organized as layers. (Similarly in the comments)